### ISSUE TITLE
Not working with async javascript

### ISSUE THREAD

**Issue by joedolson** [joedolson/wp-accessibility#26]
Using the Async JavaScript plug-in by Autoptimize (https://autoptimize.com/), toolbar doesn't work.

**Comment by joedolson** [comment]
Since all of WP Accessibility's scripting depends on the DOM being loaded, scripts should not support `async`. Most plugins that enable async will not enable it for scripts that have the defer strategy (since those two strategies are intentionally different), so the logical fix to this is to enable the defer strategy.

Since 6.3 (2023), 'defer' has been available as a backwards-compatible loading strategy for scripts that are expected to load in the footer. All of WP Accessibility's scripts already load in the footer, so enabling this strategy should be a sound solution.
